### DISCUSSION TITLE
Preinstalled Beautify and inline CSS

### DISCUSSION THREAD

**Discussion by bblessatron225** [phcode-dev/phoenix#1657] [General]
I preferred in Brackets, when I hit the Beautify button (which since it's preinstalled with Pheonix, should appear in the sidebar on the right along with live preview, etc.) that it doesn't move all the inline CSS to their own separate lines - just a waste of space! And then how the end parenthesis gets its own line also? Annoying. I had to install the beautify extension to get around this, but it would be nice if options were available so users can set their preferences.

**Comment by abose** [comment]
@bblessatron225 Can you give an example(screenshots would do about changes between the extension and built in beautify) if both the changes you would like to see with beautify, will check if that can be added.

`that it doesn't move all the inline CSS to their own separate lines` :   This happens as our integrated beautify tries to stick to line length limit of 80(will be revised to 120 in the next release). To have custom line lengths, edit preference `beautify.printWidth` to a very high value like 1000. that should inline the css too.
